Addresses the undefined attribute set to the returned WCS object by fit_wcs_from_points function (#15273)
astropy/astropy: docs/changes/wcs/15273.bugfix.rst

> Fixed a bug in ``fit_wcs_from_points`` that sets an undefined attribute to the returned WCS object.

Reviewer: ```suggestion
Fixed a bug in ``fit_wcs_from_points`` that does not set the default value of the `cdelt` of the returned WCS object.
```
I think this wording is more specific (mentions `cdelt` + it is not clear what an undefined attribute is or why it is bad to set it). My suggestion is just that. Please feel free to adjust the sentence as you wish.

Author: Thank you for your comment and I agree. The sentence has been modified as you suggested.

> Fixed a bug in ``fit_wcs_from_points`` that does not set the default value of the `cdelt` of the returned WCS object.

Reviewer: ```suggestion
Fixed a bug in ``fit_wcs_from_points`` that does not set the default value of the ``cdelt`` of the returned WCS object.
```
